index.d.ts

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+export function classNames(...args: any[]): string;
+
+export function isStatelessComponent(component: React.ComponentClass): boolean;
+
+export function getDisplayName(component: React.ComponentClass): string;
+
+export const canUseDOM: boolean;
+
+export class autoBind {
+  static react(el: React.ReactNode, options?: any): React.ReactNode;
+}
```

Comment:
This doesn't look correct. I think it should be:

```ts
export function autoBind(el: React.ReactNode, options?: any): React.ReactNode;
```

Reply:
You are completely correct. I was confused by https://github.com/sindresorhus/auto-bind#autobindreactself-options